src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -317,55 +322,71 @@ bool PolicyHandler::PolicyEnabled() const {
 
 bool PolicyHandler::LoadPolicyLibrary() {
   SDL_LOG_AUTO_TRACE();
-  sync_primitives::AutoWriteLock lock(policy_manager_lock_);
-  if (!PolicyEnabled()) {
-    SDL_LOG_WARN(
-        "System is configured to work without policy "
-        "functionality.");
-    policy_manager_.reset();
-    return false;
-  }
+  ExchangePolicyManager(nullptr);
```

Comment:
@IGapchuk shouldn't we check if Policy enabled or not before calling this?

Reply:
@LitvinenkoIra Replaced in the b2c140b69b9b991bb55f761ffb848301cf014c26